src/router/routes/mobile.js

```diff
@@ -125,6 +126,11 @@ export default [{
   path: '/browser',
   component: Apps,
   beforeEnter: ensureLoggedIn,
+}, {
+  name: 'redeem',
+  path: '/redeem',
```

Comment:
Should we add `/transfer` prefix to keep Transfer button highlighted when Redeem page opened?